src/electionguard-ui/ElectionGuard.UI/Helpers/ElectionRecordGenerator.cs

```diff
@@ -0,0 +1,132 @@
+﻿using System.IO.Compression;
+using System.Text.Json;
+using CommunityToolkit.Mvvm.DependencyInjection;
+using ElectionGuard.ElectionSetup;
+using ElectionGuard.UI.Lib.Services;
+using MongoDB.Driver;
+
+namespace ElectionGuard.UI.Helpers;
+
+public static class ElectionRecordGenerator
+{
+    private static readonly string GUARDIAN_FOLDER = "guardians";
+    private static readonly string DEVICE_FOLDER = "encryption_devices";
+    private static readonly string BALLOT_FOLDER = "submitted_ballots";
+
+    private static readonly string GUARDIAN_PREFIX = "guardian_";
+    private static readonly string DEVICE_PREFIX = "device_";

```

Comment:
These should be compile-time constants... won't see a big performance boost, but these also aren't calculated fields.

Reply:
cannot be since they are static.